.github/workflows/deploy-docs.yml

```
@@ -194,4 +208,24 @@ jobs:
+        uses: marocchino/sticky-pull-request-comment@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.CONSENT_GIT_TOKEN }}
+        with:
+          header: c15t-docs-preview
+          number: ${{ github.event.pull_request.number }}
+          message: |
+            🟡 Documentation deploy skipped.
+
+            Reason: No docs changes and upstream template unchanged, and base branch is `${{ github.base_ref }}` (not `main` or `canary`).
+            📦 Source: branch `${{ github.ref_name }}` @ `${{ github.sha }}`
             🧭 Trigger: `${{ github.event_name }}`
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)